<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>972</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>voxelToy</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMaximumSize</enum>
      </property>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="GLWidget" name="glWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>972</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="cameraDockingWidget">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Camera Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="CameraPropertiesUI" name="cameraProperties"/>
  </widget>
  <widget class="QDockWidget" name="renderingDockingWidget">
   <property name="features">
    <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>Render Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="RenderPropertiesUI" name="renderProperties"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>GLWidget</class>
   <extends>QWidget</extends>
   <header>glwidget.h</header>
   <container>1</container>
   <slots>
    <signal>shadersReloaded()</signal>
    <slot>reloadShaders()</slot>
    <slot>cameraFStopChanged(QString)</slot>
    <slot>cameraFocalLengthChanged(QString)</slot>
    <slot>cameraFocalDistanceChanged(QString)</slot>
    <slot>cameraLensModelChanged(bool)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>CameraPropertiesUI</class>
   <extends>QWidget</extends>
   <header>ui/camerapropertiesui.h</header>
   <container>1</container>
   <slots>
    <signal>onLensModelChanged(bool)</signal>
    <slot>lensModelChanged(bool)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>RenderPropertiesUI</class>
   <extends>QWidget</extends>
   <header>ui/renderpropertiesui.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
